ajax/libs/kurento-client/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "kurento-client",
+  "description": "Media API for the Kurento Web SDK",
+  "version": "6.6.0",
+  "license": "Apache-2.0",
+  "filename": "kurento-client.min.js",
+  "keywords": [
+    "API",
+    "Kurento",
+    "WebRTC"
+  ],
+  "author": "Kurento <info@kurento.com> (https://www.kurento.org)",
+  "homepage": "https://www.kurento.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Kurento/kurento-client-js.git"
+  },
+  "npmName": "kurento-client",
+  "npmFileMap": [
+    {
+      "basePath": "src/main/resources/META-INF/resources/js",
```

Comment:
I don't think it's good to use it as basePath. How about we transfer to fetch the files from https://github.com/Kurento/kurento-client-bower?